fix: UIG-2732 - vl-cascader - native & react specifieke verbeteringen toegevoegd

Goldflow opened the pull request:
[bamboo](https://www.milieuinfo.be/bamboo/browse/UIGOV-CUWC147)

[vl-cascader UIG-2732](https://www.milieuinfo.be/jira/browse/UIG-2732)
- er was een issue met Lit lifecycle in React & native html (meerbepaald voor het programmatorisch instellen van data)
- kleine verbeteringen toegevoegd (getter & setter voor `items` loste het probleem op)
- playground voorbeelden rechtgetrokken naar laatste story van cascader

kspeltix reviewed (commented):
waarom noemen de bestanden onder de playgrounds 'stories-util'
-> dat zijn toch geen Storybook stories ?
-> ik zou ze gewoon noemen: vl-cascader.data.ts / vl-cascader.utils.ts (met daarin dan de item-list functie) / vl-cascader.templates.ts

Goldflow committed:
fix: UIG-2732 - vl-cascader - native & react specifieke verbeteringen toegevoegd

Playground voorbeelden verbeterd en rechtgetrokken naar storybook voorbeelden.
Typing verbeterd.

ksvlaanderen reviewed (commented):
Top!

kspeltix commented:
:tada: This PR is included in version 1.20.0 :tada:

The release is available on [GitHub release](https://github.com/milieuinfo/uigov-web-components/releases/tag/v1.20.0)

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
